apps/www/data/PricingAddOnTable.json

```diff
@@ -8,6 +8,25 @@
       "Additional Disk Bandwidth"
     ],
     "rows": [
+      {
+        "columns": [
+          { "key": "plan", "title": "Plan", "value": "Nano (Free)" },
+          { "key": "pricing", "title": "Price USD", "value": "$0" },
+          { "key": "cpu", "title": "CPU", "value": "Shared" },
+          { "key": "dedicated", "title": "Dedicated", "value": false },
+          { "key": "memory", "title": "Memory", "value": "Up to 0.5 GB" },
+          {
+            "key": "directConnections",
+            "title": "Connections: Direct",
+            "value": "60"
+          },
+          {
+            "key": "poolerConnections",
+            "title": "Connections: Pooler",
+            "value": "200"
+          }
+        ]
+      },
```

Comment:
This mirrors the Nano (Free) from the Compute Add-ons page: https://supabase.com/docs/guides/platform/compute-add-ons